plugin/pkg/scheduler/factory/factory.go

```diff
@@ -320,9 +320,14 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 		return nil, err
 	}
 
+	predicateMetaProducer, err := f.GetPredicateMetadataProducer()
+	if err != nil {
+		return nil, err
+	}
+
 	f.Run()
 
-	algo := scheduler.NewGenericScheduler(f.schedulerCache, predicateFuncs, priorityMetaProducer, priorityConfigs, extenders)
+	algo := scheduler.NewGenericScheduler(f.schedulerCache, predicateFuncs, priorityMetaProducer, predicateMetaProducer, priorityConfigs, extenders)
```

Comment:
i think probably if thats the case ill reorg the prioconfigs to be one arg earlier to, so that the types are after the configs